editor/actions.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+export function savePost( dispatch, post ) {
```

Comment:
I would prefer to address this separately as well.  Unlike other parts of this PR (how we actually store the post + its edits in state), it should be pretty easy to refactor this later without touching lots of other parts of the code.